src/hotspot/share/utilities/globalDefinitions.cpp

```diff
@@ -159,7 +159,7 @@ void basic_types_init() {
   static_assert(is_power_of_2(HeapWordSize), "HeapWordSize must be power of 2");
   static_assert((size_t)HeapWordSize >= sizeof(juint),
                 "HeapWord should be at least as large as juint");
-  static_assert(sizeof(NULL) == sizeof(char*), "NULL must be same size as pointer");
+  static_assert(sizeof(nullptr) == sizeof(char*), "nullptr must be same size as pointer");
```

Comment:
Adding in another static_assert here.

Reply:
Not sure this makes any sense for `nullptr`.

Reply:
Yeah, I'm removing this static assert.